src/processors/BaseProcessor.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "JuceProcWrapper.h"
+#include "netlist_helpers/CircuitQuantity.h"
```

Comment:
It would be great if we didn't need to include this inside this header... I'm curious what happens if we remove it?

Reply:
Hey Jatin, I included this header as I was getting errors saying that the type was incomplete (having only the forward declaration in BaseProcessor) removing the include introduces the error `error: invalid application of 'sizeof' to an incomplete type 'netlist::CircuitQuantityList'` let me know if there is a better way around this!